xia_api/__init__.py,sha256=ovKktqqog0rw5SkqYhewq0cHtGFSyr76gIH_XpCITSo,485
xia_api/auth_client.cpython-310-x86_64-linux-gnu.so,sha256=i5kncuOUG72dklh5mDVCXTg3maAPpPf3SPOk9GRJsck,743880
xia_api/message.cpython-310-x86_64-linux-gnu.so,sha256=J4Dnac1J6peJW8bWO3IUQ3erNH-RFeJLTmQDdi0fyBM,780776
xia_api/open_api.cpython-310-x86_64-linux-gnu.so,sha256=yZxxZ_771ySLSn5AuUgQbgD6Ew7lhC2gl9gCZwliyOk,1034728
xia_api/rest.cpython-310-x86_64-linux-gnu.so,sha256=lH2mSZfFMJrym9JOZd8aG9QNa6Bz8mcPvm6tXdNjNjQ,891432
xia_api-0.0.42.dist-info/LICENSE.txt,sha256=Ttf2VW9fqSQdbOE-h8EVivyg_c5obqacx70-ob7oLI4,155
xia_api-0.0.42.dist-info/METADATA,sha256=46-JIkA9EzK-1NLUFIDmZw20dgzvRB4eg4mTkEF0QAs,641
xia_api-0.0.42.dist-info/WHEEL,sha256=Kn0XEVEprF3m5Ls7p1vDRSgmlko_ccaWbbQdh4VEEUE,108
xia_api-0.0.42.dist-info/top_level.txt,sha256=krL7__RbzvbaNfgnuF-oCqAnr2y3Yy6whD8CBZyVXyw,8
xia_api-0.0.42.dist-info/RECORD,,
